{{cookiecutter.repo_name}}/pyproject.toml

```
@@ -0,0 +1,49 @@
+[build-system]
+requires = [
+  "setuptools >= 40.9.0",
+  "wheel",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{cookiecutter.repo_name}}"
+description = "{{cookiecutter.description}}"
+license = {{ "{" }}file = "LICENSE" {{ "}" }}
+authors = [
+    {{ "{" }}name = "{{cookiecutter.author_name}}", email = "{{cookiecutter.author_email}}"{{ "}" }},
+]
+maintainers = [
+    {{ "{" }}name = "{{cookiecutter.author_name}}", email = "{{cookiecutter.author_email}}"{{ "}" }},
+]
+readme = "README.md"
+requires-python = ">=3.8"
+dependencies = [
+    "MDAnalysis>=2.0.0",
+]
+keywords = [
+    "molecular dynamics",
```

```suggestion
    "molecular simulations",
```
Suggestion purely based on the idea that some folks might not use this for MD work (I'm thinking the docking, etc... folks out there)